x-pack/plugins/secops/public/components/draggables/index.test.tsx

```diff
@@ -31,7 +31,7 @@ describe('draggables', () => {
     test('it renders the default Badge', () => {
       const wrapper = shallowWithIntl(
         <DraggableBadge
-          id="draggable-id"
+          eventId="draggable-id"
```

Comment:
Given that the draggable id is now derived from the `eventId`, and other props, consider changing `draggable-id` to something like `event-id` in this test, and related tests in this pr, and the suite of open row renderer PRs [33467](https://github.com/elastic/kibana/pull/33467) [33469](https://github.com/elastic/kibana/pull/33469) [33470](https://github.com/elastic/kibana/pull/33470)

Reply:
I think `draggable-id` is still clear enough for the reader and works just as well if it's ok with you. It is still a type of id that is draggable.

I will keep it for now.